include/vcpkg/build.h

```diff
@@ -204,17 +204,33 @@ namespace vcpkg
 
     struct BuildResultCounts
     {
-        int succeeded = 0;
-        int build_failed = 0;
-        int post_build_checks_failed = 0;
-        int file_conflicts = 0;
-        int cascaded_due_to_missing_dependencies = 0;
-        int excluded = 0;
-        int cache_missing = 0;
-        int downloaded = 0;
-        int removed = 0;
-
-        void increment(const BuildResult build_result);
+        unsigned int succeeded = 0;
```

Comment:
Be careful that callers of this don't need to change `%d` to `%u` or similar.

Reply:
They are not used that way